src/github-action/index.ts

```diff
@@ -0,0 +1,17 @@
+import * as core from "@actions/core";
+
+import { VersionResolver } from "../version";
+
+import { newCliInstaller } from "./cli-installer";
+
+// Installs the 1Password CLI on a GitHub Action runner.
+export const installCliOnGithubActionRunner = async (
+	version?: string,
+): Promise<void> => {
+	const versionResolver = new VersionResolver(
+		version ?? core.getInput("version"),
+	);
+	await versionResolver.resolve();
+	const installer = newCliInstaller(versionResolver.get());
+	await installer.installCli();
```

Comment:
`[nit]`

Would it look better if we do something like this?
```suggestion
	await newCliInstaller(versionResolver.get()).installCLI();
```

Reply:
It wants create an instance of the class before accessing its method. So `await newCliInstaller(versionResolver.get()).installCLI();` won't compile. Leaving as it is now.